.eslintrc.yml

```diff
@@ -17,3 +17,7 @@ globals:
   GMaps: true
   Fuse: true
   instantsearch: true
+  CMS: true
+  createClass: true # Global from Netlify CMS
+  h: true
+  marked: true
```

Comment:
Please add a comment that `h` and `marked` are only for the CMS module and comment what `h` is and where it's defined, otherwise other contributors will likely have no idea why these are globals and what there purpose is...

Reply:
Done.